cloudflared/rootfs/etc/cont-init.d/10-cloudflared-config.sh

```diff
@@ -138,12 +138,13 @@ hasTunnel() {
 
     bashio::log.info "Existing tunnel with ID ${tunnel_uuid} found"
 
-    # Check if tunnel name in file matches config value
+    # Get tunnel name from Cloudflare API by tunnel id and chek if it matches config value
     bashio::log.info "Checking if existing tunnel matches name given in config"
-    local tunnel_name_from_file
-    tunnel_name_from_file="$(bashio::jq "${data_path}/tunnel.json" .TunnelName)"
-    bashio::log.debug "Tunnnel name read from file: $tunnel_name_from_file"
-    if [[ $tunnel_name != "$tunnel_name_from_file" ]]; then
+    local existing_tunnel_name
+    existing_tunnel_name=$(cloudflared --origincert="${data_path}/cert.pem" tunnel \
+        list --output="json" --id="${tunnel_uuid}" | jq -er '.[].name')
```

Comment:
I think you can use `bashio::jq` here, no?

Reply:
```suggestion
        list --output="json" --id="${tunnel_uuid}" | bashio::jq -e '/dev/stdin' '.[].name')
```

Not sure, but it may work.

Reply:
Nevermind. I think it does not accept the -e.

Reply:
I also tried around with bashio::jq, I guess this would work, but not sure.
```
bashio::jq "$(cloudflared --origincert="${data_path}/cert.pem" tunnel \
    list --output="json" --id="${tunnel_uuid}")" '.[].name'
```
I think if we want to add more features in the future, we'll have to restructure all the code either way. Things getting confusing when working with only one file